terms/live-edit-and-locked-edits.md

```diff
@@ -2,6 +2,8 @@
 
 > “Data that provides information about other data” \[[wikipedia](https://en.wikipedia.org/wiki/Metadata)]
 
+Metadata is defined as data about data. It gives context to the data it describes. Although this is a complete definition of what metadata is, it is not specific enough for most people. So, it can be useful to refine the idea of metadata to more specific subsets: [schemas][schema.md], [vocabularies][vocabulary.md], and ontologies[ontology.md].
```

Comment:
"Metadata is defined as data about data." is an almost direct repeat of the main definition line above (“Data that provides information about other data” with its citation).

I think the comment "Although this is a complete definition of what metadata is, it is not specific enough for most people." is probably too subjective to be useful here.

Finally, whilst a schema, vocabulary or ontology reference might be one *part* of the metadata associated with a data object, and whilst an instance in an ontology would *contain* metadata, these things are far from being conceptually similar to metadata in their own right, so I think this is very confusing to be honest.

Reply:
I can't think of any metadata that can't be classified as vocabulary, schema, or an ontology. Curious if you have an example in mind.